src/offset/local/unix.rs

```diff
@@ -162,10 +150,11 @@ impl Cache {
             (Source::Environment { .. }, Source::LocalTime { .. })
             | (Source::LocalTime { .. }, Source::Environment { .. }) => true,
             // stay as file, but mtime has changed
-            (Source::LocalTime { mtime: old_mtime }, Source::LocalTime { mtime })
-                if old_mtime != mtime =>
-            {
-                true
+            (Source::LocalTime, Source::LocalTime) => {
```

Comment:
Why is this better? It seems less precise to me.

Reply:
Hmm, maybe I should keep an mtime around. It is not guaranteed to be the real modification time. We should either use ctime or keep track of the mtime.

Comparing ctime to `last_checked` is simpler though.